templates/bosh-lite-manifest-template-v1.yml

```diff
@@ -92,6 +92,7 @@ jobs:
     properties:
       api_server: (( api_server_properties ))
       policy_db: (( policydb ))
+      cf: (( cf_properties ))
```

Comment:
as the apiserver configuration only requires cf api. 
So it should be something like this.
```yaml
cf:
   api: (( cf_properties.api ))
```